generators/php/codegen/src/asIs/Constant.Template.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace <%= namespace%>;
+
+use DateTimeInterface;
+
+class Constant
+{
+    public const DateFormat = 'Y-m-d';
+    public const DeserializationDateFormat = "!" . self::DateFormat;
+    public const DateTimeFormat = DateTimeInterface::RFC3339;
```

Comment:
I noticed that we _might_ need to export an additional constant here when we want to serialize the DateTIme as a `string` - it looks like `DateTimeInterface::RFC3339` might not work in all cases (re: https://github.com/fern-api/fern/pull/4655):

```php
            $query['deadline'] = $request->deadline->format('c');
```

Not sure what we should name this given that we need to use this `DateTimeFormat` elsewhere.

Reply:
talked offline, it does seem to work as is!